lang/ui.en.json

```diff
@@ -731,6 +731,34 @@
     "defaultMessage": "Default (red, blue, green)",
     "description": "Graph colour scheme option"
   },
+  "graph-line-scheme": {
+    "defaultMessage": "Graph line style",
+    "description": "Graph line scheme setting label"
+  },
+  "graph-line-scheme-accessible": {
+    "defaultMessage": "Accessible lines (solid, dashed, dots)",
+    "description": "Graph line scheme option"
+  },
+  "graph-line-scheme-solid": {
+    "defaultMessage": "Solid lines",
+    "description": "Graph line scheme option"
+  },
+  "graph-line-weight": {
+    "defaultMessage": "Graph line thickness",
+    "description": "Graph line weight setting label"
+  },
+  "graph-line-weight-normal": {
+    "defaultMessage": "Normal",
```

Comment:
Maybe a more neutral "Medium" ?

Reply:
Also unsure about the value of "Thin"...

Reply:
Light?

Reply:
Change for normal -> medium [b1acf11](https://github.com/microbit-foundation/ml-trainer/pull/548/commits/b1acf11707c6ef853268caee6c1bc170ab87b90d).

Reply:
After discussion let's drop Thin and have Default (as per before) and Thick (as you think!).
No view on whether we slightly tweak the recording graph lines.

Reply:
Done here [586de87](https://github.com/microbit-foundation/ml-trainer/pull/548/commits/586de871619c5c3b4e053d2e30402edd837429a7)